travel limited to students and postdocs (#8)
ProjectPythia/pythia-cookoff-2025: faq.md

## Are there travel funds available?

> Participant Travel Support (PTS) will be made available for a limited number of attendees. Participant travel support is only available to students and PostDocs residing in the US. Please indicate your support needs when you [register](https://www.eventsquid.com/contestant-reg.cfm?event_id=27191). Applications for PTS are due by June 20. 

Reviewer: ```suggestion
Participant Travel Support (PTS) will be made available for a limited number of student and PostDoc attendees who are residing in the US. Please indicate your support needs when you [register](https://www.eventsquid.com/contestant-reg.cfm?event_id=27191). Applications for PTS are due by June 20. 
```
